### ISSUE TITLE
Excuse me 

### ISSUE THREAD

**Issue by figuifreedos** [Wolfgange3311999/Mine-DS#80]
Excuse me, I'm having trouble installing Mine ds fat on dsi with Twilight menu, could someone explain to me how to install it please?‽

**Comment by MatthewScholefield** [comment]
Yeah, just extract the zip file somewhere, copy the NDS file anywhere on the SD card and also copy the contents of the "SD card" folder directly onto the SD card.

If twilight menu crashes it may still work if you launch via HBmenu (Ie. Twilight menu -> HBMenu -> Mine DS).

**Comment by figuifreedos** [comment]
thank you so much
